engines/instance_verification/lib/instance_verification/engine.rb

```diff
@@ -58,8 +58,9 @@ def verify_product_activation
 
           if product.base?
             verify_base_product_activation(product)
-          else
-            verify_extension_activation(product)
+          elsif !product.free? && @system.payg?
```

Comment:
shouldn't we just ask if product is free ?

Meaning a `hybrid` system may have extensions included in its subcription and may does not need to call SCC for that product ?